Makefile

```diff
@@ -36,6 +36,7 @@ PKG          := ./...
 TAGS         :=
 TESTS        := ".*"
 TESTTIMEOUT  := 15s
+CPUS         := 1
```

Comment:
I'm pretty sure `-cpu` only affects benchmarks. And yes, it will run the benchmarks once for each cpu specification, appending `-${GOMAXPROCS}` to the benchmark name in the output.